### PR TITLE
Add exercise: leap

### PR DESCRIPTION
**Pull request by andrej-makarov-skrt**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -5,7 +5,13 @@
   "active": false,
   "test_pattern": "TODO",
   "exercises": [
-
+    {
+      "slug": "leap",
+      "difficulty": 1,
+      "topics": [
+        "generators"
```
Comment (SaschaMann) [inline config.json]: The example solution doesn't have any generators, perhaps change this to `control-flow (if-else statements)` or `comparisons` or similar?

**Comment by SaschaMann** [comment]
Good idea to add the exercise text in the example! I just added a minor comment about the config.

File: config.json (hunk 1 above)
Comment (andrej-makarov-skrt) [inline config.json]: @SaschaMann yes, I agree, just used this from your pull request.
Are these three okay: Control-flow (conditionals), Integers, Mathematics?

**Comment by andrej-makarov-skrt** [comment]
I close this due to wrong actions from my side.

File: config.json (hunk 1 above)
Comment (andrej-makarov-skrt) [inline config.json]: @SaschaMann I made a new one pull request, please check it out.

**Pull request closed without merging**